FreeRTOS-Plus/Demo/MQTT_Mutual_Auth_Qemu_mps2/main_networking.c

```diff
@@ -0,0 +1,325 @@
+/*
```

Comment:
Is this different from main_networking.c in the FreeRTOS_Plus_TCP_Echo_Qemu_mps2 demo?

Reply:
Yes